.gitignore

```diff
@@ -13,7 +13,7 @@ build
 !.idea/icon.svg
 out
 *.iml
-.vscode
+.vscode`
```

Comment:
Is it a mistake?

Reply:
It looks like this PR needs to be rebased onto the latest main